script/install.sh

```diff
@@ -903,6 +903,7 @@ write_env_vars() {
     echo "    unset -f deactivate"
     echo "}"
     echo
+    echo "[[ -v _OLD_D_PATH ]] && deactivate"
```

Comment:
Looks like this is needed for OSX:

```suggestion
    echo "if [ -v _OLD_D_PATH ] ; then deactivate; fi"
```

Reply:
Works on my osx, but better safe than sorry.